pysages/methods/ann.py

```diff
@@ -286,8 +286,12 @@ def analyze(result: Result[ANN]):
     _, layout = unpack(model.parameters)
 
     def build_fes_fn(nn):
-        params = pack(nn.params, layout)
-        return jit(lambda x: nn.std * model.apply(params, x))
+        def fes_fn(x):
+            params = pack(nn.params, layout)
+            A = nn.std * model.apply(params, x) + nn.mean
+            return A.max() - A
```

Comment:
Hmm, shouldn't this be `A-A.min()` to have the lowest point to be zero?

Alternative, what I also can see as reasonable is `A - A[0]` to ensure that it always starts at 0.
That makes sense for integrated free-energies for example, where you say the integration constant is 0.

Reply:
The network is trained against the negative of the free energy, so `A.max() - A` is correct here. For now I would keep everything as we have and decide in favor another option in the future.